scripts/content.js

```diff
@@ -87,22 +87,33 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
 
                   var TransparencyScoret=document.createElement('span');
                   TransparencyScoret.classList.add('scoret');
-                  TransparencyScoret.innerText=" TransparencyScore "
+                  TransparencyScoret.innerText=" Transparency Rate "
 
 
                   //EnvironmentScore
                   var ES=document.createElement('div');
                   ES.classList.add('es');
 
-                  var EnvironmentScore=document.createElement('span');
-                  EnvironmentScore.classList.add('score');
-                  EnvironmentScore.innerText=resp.result[index].EnvironmentScore;
-                  EnvironmentScore.style["background-color"] =resp.colourMap[resp.result[index].EnvironmentScore];
+
+                  if(resp.result[index].EnvironmentScore){
```

Comment:
this is a lot of unnecesary repeated code, the only thing that is different is the `innerText` - although should the `background-color` change as well?